File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {
        const map = new Set(nums);
        let count=0;
        for(let i of nums){
            if(!map.has(i-1)){
                let c=i;
                let s=1;
                while(map.has(c+1)){
                    c+=1;
                    s+=1;
                }
                count = Math.max(count,s)
            }
        }
        return count;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
        int ca=0;
        int count=0;
        for(int num:nums){
            if(count==0){
                ca=num;
            }
            if(num==ca){
                count++;
            }
            else{
                count--;
            }
        }
        return ca;
    }
}